### DISCUSSION TITLE
Creation of files to be uploaded to Packet Sender

### DISCUSSION THREAD

**Discussion by lsalgueroh** [dannagle/PacketSender#330] [Q&A]
Hello!

I'm working with an automata for which I want to send many frames via TCP varying their value. Seeing the option to "load a file" I thought that there could be a possibility to create a file with different frames, load it, and the PacketSender itself will send them one by one. However, I have not been able to get it to work as I expected.

Is it possible to do this? If so, what type of file? .text? And how should I separate the frames so that Packet Sender recognises them as independent frames? 

Thank you very much in advance!

Laura :)

**Comment by dannagle** [comment]
The "load file" is for any file in any format. Packet Sender will read it in as the data to be sent for that TCP session.

Packet Sender does not offer Layer 2 customization where the actual TCP frames are generated. Packet Sender will load the entire file (up to around 10K in the GUI, 100M from the command line) and then let your OS handle the details of segmenting, frames, etc, as it sends it out.
